decidim-core/app/cells/decidim/collapsible_list/show.erb

```diff
@@ -15,14 +15,14 @@
       <span class="collapsible-list__see-less" data-toggle="collapsible-list-<%= seed %>"><%= t("decidim.collapsible_list.see_less") %></span>
     </span>
   </div>
-<% else %>
-  <div class="collapsible-list <%= list_class %>">
-    <% list.each do |element| %>
-      <% if cell_name %>
-        <%= cell cell_name, element, cell_options %>
-      <% else %>
-        <%= card_for element, cell_options %>
+  <% else %>
+    <div class="collapsible-list<%= list_class %>">
```

Comment:
Mind that this generates the following HTML: 
```html
    <div class="collapsible-listrow small-up-1 medium-up-2 card-grid">
```

What is doing is that it's actually creating a new non-existing class (`.collapsible-listrow`), while just removing the `collapsible-list` class should be enough. This is what I have in mind: 
```suggestion
    <div class="<%= list_class %>">
```

Can you change that please?